td4m3/src/main/java/ca/td/td4m3/mortgage/view/MortgageActivityView.kt

```diff
@@ -1,4 +1,5 @@
 package ca.td.td4m3.mortgage.view
 
 interface MortgageActivityView {
+    fun getMortgageCalculatorActivity()
```

Comment:
shouldn't this be launchMortgageActivityView since you'll be launching from here?

Reply:
No, this will launch the calculator activity, from the MortgageActivity.

Reply:
Yeah so shouldn't the name start with "launch" and not get? Like what you're saying is what I'm saying.

Reply:
Yeah so shouldn't the name start with "launch" and not get? Like what you're saying is what I'm saying.

Reply:
Renamed more appropriately for both getting the listener and launching the activity.